apps/SageAccountWeb/src/components/TwoFactorAuth/ResetTwoFactorAuth.tsx

```diff
@@ -40,14 +42,19 @@ export function ResetTwoFactorAuth() {
     Pick<TwoFactorAuthErrorResponse, 'twoFaToken' | 'userId'> | undefined
   >()
 
-  const promptForSignIn = token && !twoFaRequiredToken
-  const promptConfirmDisable2FA = token && twoFaRequiredToken
+  const [password, setPassword] = useState('')
+  const [showPasswordField, setShowPasswordField] = useState(false)
```

Comment:
When resetting 2FA with a password, we prompt for password at the same time as submitting the 2FA disable request. That request will fail if the password is wrong, so the user can recover and fix the incorrect password using this UI